From feef137a1fcebf0653fa3baccf211fa616f49112 Mon Sep 17 00:00:00 2001
From: Kyle Steere <kyle.steere@chainguard.dev>
Date: Mon, 14 Jul 2025 10:32:52 -0500
Subject: [PATCH] update commons-lang3 version update to 3.18

Patch updated to bump io.netty to remediate GHSA-3p8m-j85q-pgmj

Signed-off-by: Kyle Steere <kyle.steere@chainguard.dev>
---
 .build/parent-pom-template.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/.build/parent-pom-template.xml b/.build/parent-pom-template.xml
index a3f507706c..2de8f0a45c 100644
--- a/.build/parent-pom-template.xml
+++ b/.build/parent-pom-template.xml
@@ -359,7 +359,7 @@
       <dependency>
         <groupId>org.apache.commons</groupId>
         <artifactId>commons-lang3</artifactId>
-        <version>3.13.0</version>
+        <version>3.18.0</version>
       </dependency>
       <dependency>
         <groupId>org.apache.commons</groupId>
@@ -422,7 +422,7 @@
       <dependency>
         <groupId>com.fasterxml.jackson.core</groupId>
         <artifactId>jackson-core</artifactId>
-        <version>2.13.2</version>
+        <version>2.15.0</version>
       </dependency>
       <dependency>
         <groupId>com.fasterxml.jackson.core</groupId>
@@ -729,7 +729,7 @@
       <dependency>
         <groupId>io.netty</groupId>
         <artifactId>netty-all</artifactId>
-        <version>4.1.96.Final</version>
+        <version>4.1.125.Final</version>
         <exclusions>
           <exclusion>
             <groupId>io.netty</groupId>

-- 
2.43.0
